docs/generate_genome_data.md

```diff
@@ -0,0 +1,66 @@
+# Generate Genome Collection Data
+
+## Generate Genome Attributes
+
+This section describes the steps and scripts used to generate attributes of genomes. 
+The scripts used in the process are located in the `src/loaders` directory.
+
+1. Retrieve Source Genome Files
+   * From NCBI
+      * Download the genome files from the NCBI FTP server using the script 
+      `ncbi_downloader/ncbi_downloader.py`. 
+   * From workspace
+     * `TODO: work remains to be done`
+2. Compute Genome Attributes
+   * The next step is to execute tools to compute various attributes of the genome using the script 
+   `genome_attributes/compute_genome_attribs.py`. 
+   * This script currently supports two tools for attribute computation:
+   [GTDB-TK](https://ecogenomics.github.io/GTDBTk/index.html) 
+   and [CheckM2](https://github.com/chklovski/CheckM2)
+3. Parse Tool Results
+   * After the attribute computation tools have been executed, the results need to be parsed and organized 
+   into a format that is suitable for importing into ArangoDB. This is done using the script 
+   `genome_attributes/parse_computed_genome_attribs.py`. 
+4. Import JSON Format File Into ArangoDB
+   ```commandline
+   # set up an SSH tunnel
+   USER_NAME=user_name  # user name for login1.berkeley.kbase.us
+   FORWARD=localhost:48000
+   ssh -f -N -L $FORWARD:10.58.1.211:8531 \
+    $USER_NAME@login1.berkeley.kbase.us
+   
+   # execute arangoimport
+   PARSED_FILE=json_file_path    # the file path generated by the parsing script
+   ARANGO_USER=collections_dev   # arangoDB user name
+   ARANGO_PW=arango_password     # arangoDB password
+   arangoimport --file $PARSED_FILE \
+    --server.endpoint tcp://$FORWARD \
+    --server.username $ARANGO_USER \
+    --server.password $ARANGO_PW \
+    --server.database collections_dev \
+    --collection kbcoll_genome_attribs
+   ```
+
+
+Note:
+* For usage instructions for each script, please refer to the help option (-h) of the script.
+* Default directory layout at NERSC
+  * source genome files: `/global/cfs/cdirs/kbase/collections/sourcedata/[kbase_collection]/[load_ver]`
+  * computed genome files: `/global/cfs/cdirs/kbase/collections/collectionsdata/[kbase_collection]/[load_ver]/[tool_name]`
+
+## Generate Genome Taxonomy Ranks
+
+This section outlines the procedures and tools utilized to determine the taxonomic ranks, in order of hierarchy, 
+for the genome taxa counts.
+
+1. Retrieve Taxonomy File
+   * From GTDB
+     * Download taxonomy files (e.g. `ar53_taxonomy_r207.tsv`) from [GTDB](https://data.gtdb.ecogenomic.org/)
+   * Using computational tools such as GTDB-TK
+     * Please refer to the "Generate Genome Attributes" section above and follow the steps 1, 2, 3 as described.
+2. Prepare Taxa Count Data
+   * Process the GTDB taxonomy file by utilizing the script `gtdb/gtdb_taxa_count_loader.py`. 
+   * Process the JSON file parsed from computational tools by using the script `TODO: to be implemented`
```

Comment:
I think this script should be able to handle both the output of the gtdb metadata parser and the tool parser right? It should look more or less the same

Reply:
👍

Reply:
:+1: